.github/dependabot.yml

```diff
@@ -8,7 +8,6 @@ updates:
     labels:
       - automation
       - dependency
-      - Team:Fleet
```

Comment:
I think you want to remove `reviewers:` as `elastic/fleet` just below as well. If you completely remove that section the code owners file will hopefully take precedence.

Reply:
Done in fcda4127f471621e267c6018b741a4223288af07 - thanks for flagging.